src/mongo/commands.ts

```diff
@@ -74,6 +75,13 @@ export class MongoCommands {
 		}
 	}
 
+	private static validateCollectionName(key: string): string | undefined | null {
```

Comment:
This exact same logic should apply to DocDB

Reply:
There are mongo specified restrictions on names ([specified here for eg](https://docs.mongodb.com/manual/reference/limits/#naming-restrictions)), which could be different from Cosmos/DocumentDB's naming ([this page has some rules](https://docs.microsoft.com/en-us/azure/cosmos-db/create-documentdb-dotnet#add-a-collection)). I don't yet see why they would have the same logic on our side.